tsconfig.path.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "usTaxes/*": ["src/*"]
```

Comment:
is `us-taxes` or `ustaxes` more natural? This is kind of like an npm lib name, like `material-ui` or https://github.com/ustaxes/UsTaxes/blob/6ca9ad6a117b1c63ee8b29846e0dc21b5a735cb7/src/redux/store.ts#L6